src/main/java/com/github/tomakehurst/wiremock/matching/EqualToXmlPattern.java

```diff
@@ -45,6 +45,7 @@ public class EqualToXmlPattern extends StringValuePattern {
           SCHEMA_LOCATION,
           NO_NAMESPACE_SCHEMA_LOCATION,
           NODE_TYPE,
+          NAMESPACE_PREFIX,
```

Comment:
Do we actually care about whether the prefix is the same?

My understanding is that as long as two prefixes refer to the same URI they're semantically equivalent.

Reply:
i think it depends on what the user wants. i figure putting it in will allow the user to do prefix comparison but also allow them to opt out via the exemptedComparisons attributes if they're not interested. that way they can configure their matcher how they please

Reply:
This could end up breaking a lot of folks' tests for something that's not a meaningful difference. I suggest we pull this one out for the time being, or at least make it opt-in rather than out.

Reply:
done